metrics/otel_metrics.go

```diff
@@ -189,7 +189,7 @@ func (o *OTelMetrics) Register(name string, metricType string) {
 	case "counter":
 		ctr, err := o.meter.Int64Counter(name)
 		if err != nil {
-			o.Logger.Error().WithString("msg", "failed to create counter").WithString("name", name)
```

Comment:
Ooohhh, I'm not sure about this change. Having a field consistently called "msg" is a pattern that I at least have used repeatedly. Our log queries all use "msg" as a quick way to see what's going on in the log.

I don't entirely hate giving it a different name, but I do think we shouldn't eliminate the idea of a common field that all the logs have.

Reply:
I updated these because I don't believe the log output at all because it doesn't end with a `Logf("..")`. The 
`WithField` and `WithString` func return an `Entry` struct so it can build chained with multiple things, but does nothing until you finish with a `LogF`.

Reply:
Oh, wow, yeah, I've been burned by that before! OK, I still think that it's useful to have a low(ish)-cardinality "msg" field in our logs so you can group by msg and see spikes.

Reply:
After further discussion 1:1 - the body of the message will be low cardinality so it's easy to see the volume of a specific log message. Log attributes set using `WithField`, `WithFields` and `WithString` can contain higher cardinality values.